docs/gno-infrastructure/setting-up-a-local-chain.md

```diff
@@ -10,8 +10,59 @@ In this tutorial, you will learn how to start a local Gno node (and chain!).
 Additionally, you will see the different options you can use to make your Gno instance unique.
 
 ## Prerequisites
+- **Git**
+- **`make` (for running Makefiles)**
+- **Go 1.21+**
+- **Go Environment Setup**:
+  - Make sure `$GOPATH` is well-defined, and `$GOPATH/bin` is added to your `$PATH` variable.
 
-- [`gnoland` installed](local-setup.md#3-installing-other-gno-tools).
+## Installation
+
+To install the `gnoland` binary/tool, clone the Gno monorepo:
+
+```bash
+git clone https://github.com/gnolang/gno.git
+```
+
+After cloning the repo, go into the `gno.land/` folder, and use the existing
+Makefile to install the `gnoland` binary:
+
+```bash
+cd gno.land
+make install.gnoland
+```
+
+To verify that you've installed the binary properly and that you are able to use
+it, run the `gnoland` command:
+
+```bash
+gnoland
```

Comment:
This should be `gnoland --help`

Reply:
[60fcbfc](https://github.com/gnolang/gno/pull/2212/commits/60fcbfc98cd3d07ac8764a46e7be05e7b45fc426)